scripts/clan.py

```diff
@@ -813,7 +813,7 @@ def load_clan_json(self):
         )
         game.clan.post_initialization_functions()
 
-        game.clan.reputation = int(clan_data["reputation"])
+        game.clan.reputation = min(0, int(clan_data["reputation"]))
```

Comment:
As clan reputation is 0-100, you could use 
`game.clan.reputation = clamp(int(clan_data["reputation"]),0,100)`

Reply:
I don't see a clamp function being used anywhere else in the code and the code I put here is based on the `change_clan_relations` function below.

Reply:
Ah, my bad, it's only in development branch